pimcore/lib/Pimcore/Placeholder/ConcreteObject.php

```diff
@@ -16,7 +16,7 @@
 
 use Pimcore\Model;
 
-class Object extends AbstractPlaceholder
+class ConcreteObject extends AbstractPlaceholder
```

Comment:
Please use `DataObject` as the class name, wich is also used in v5: 
https://github.com/pimcore/pimcore/blob/c62924ca5c3d0c482a71097cb0231aff143208d6/lib/Placeholder/DataObject.php#L19